src/main/java/com/iopipe/IOpipeConfigurationBuilder.java

```diff
@@ -203,5 +207,28 @@ public final void setTimeOutWindow(int __ms)
 		
 		this._timeoutwindow = __ms;
 	}
+	
+	/**
+	 * Set to true {@code true} if cold start detection is to be managed per
+	 * individual instance of {@link IOpipeService}, this will result in the
+	 * first execution under that instance being treated as a cold start.
+	 *
+	 * Otherwise {@code false} will use cold start detection on a per process
+	 * basis.
+	 *
+	 * This generally is not needed and is not recommended as warm starts will
+	 * be flagged as coldstarts if a new instance is created. This is only for
+	 * advanced service usage.
+	 *
+	 * This defaults to {@code false}.
+	 *
+	 * @param __yes If {@code true} then coldstarts are to be managed per
```

Comment:
The variable `_localcoldstart` is initialized to its default `false` where the variables are at the top of the class. You do not have to call this method at all when initializing the configuration, it is optional. There could be a parameterless version but then you would have two methods that will be setting or clearing a boolean.

As for the method parameter I can make it `__enabled`, if that makes more sense since it would enable coldstarts managed per instance?

As for the coldstart behavior since this option defaults to off (making coldstart checks done on a global atomic boolean) it will not affect any existing code or change how coldstarts behave. The only place this setting is used is in the coldstart test since coldstarts now have auto labels (<https://github.com/XerTheSquirrel/iopipe-java/blob/05b2860686a7d8cf11e7f6178ae212afe460a966/src/test/java/com/iopipe/__DoColdStartAutoLabel__.java>). I feel adding the configuration is the fastest and cleanest solution to testing that the label is set for coldstarts.